charts/mlrun-ce/README.md

```diff
@@ -185,6 +197,23 @@ Your applications are now available in your local browser:
 > - You can add and configure a k8s ingress-controller for better security and control over external access.
 
 
+## Upgrading the Chart
+When new versions of MLRun CE are released you can upgrade your chart to the new version.
+To upgrade the chart, use the following command:
+
+```bash
+helm --namespace mlrun upgrade my-mlrun mlrun/mlrun-ce
+```
+
+### Triggering DB Migrations
```

Comment:
```suggestion
### Triggering DB Migrations

```

Reply:
also important to note that once schema is migrated, there is no worn to downgrade it